docs/dev/swagger.yaml

```diff
@@ -340,6 +396,35 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/Galley'
+  IssueSummary:
+    type: object
+    properties:
+      id:
+        type: integer
+      _href:
+        type: string
+      title:
+        $ref: '#/definitions/LocaleObject'
+      description:
+        $ref: '#/definitions/LocaleObject'
+      identification:
+        type: string
+      volume:
+        type: integer
+      number:
```

Comment:
It's not good practice, but it's possible. I suspect there are a bunch of weird variants out in the wild, e.g. `spring`, `9-10`, `3a`, etc. I'd suggest keeping it a string, odd as that seems.